Makefile

```diff
@@ -127,6 +127,20 @@ vet: gowork ## Run go vet against code.
 	go vet ./...
 	go vet ./api/...
 
+APIPATH ?= $(shell pwd)/api
+.PHONY: tidy
+tidy: fmt
+	go mod tidy; \
+	pushd $(APIPATH); \
+	go mod tidy; \
+	popd
+
+.PHONY: golangci-lint
+golangci-lint:
+	# NOTE this will install golangci-lint in to local bin dir
+	test -s $(LOCALBIN)/golangci-lint || curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.51.2
```

Comment:
should that be v1.52.2 assuming that is the golanci-lint version 

( you have that 1.52.2 at  https://github.com/openstack-k8s-operators/cinder-operator/pull/147/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9)